pkg/minikube/command/exec_runner.go

```diff
@@ -184,6 +184,24 @@ func (e *execRunner) Copy(f assets.CopyableFile) error {
 	return writeFile(dst, f, os.FileMode(perms))
 }
 
+// CopyFrom copies a file
+func (e *execRunner) CopyFrom(f assets.CopyableFile) error {
```

Comment:
lets add more explanitory comment ,how is this different from Copy func above ?

Reply:
It goes in the other direction, Copy is hard-coded to only CopyTo the VM

Reply:
It's an alternative implementation of the API #11296, the other PR looked incomplete